superagi/controllers/tool.py

```diff
@@ -72,10 +72,10 @@ def get_tool(
 def get_tools(
```

Comment:
this endpoint should be /list. get should only get one record.

Reply:
Updated.